.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "esp/src/crossfilter"]
 	path = esp/src/crossfilter
 	url = https://github.com/hpcc-systems/crossfilter.git
+[submodule "plugins/kafka/librdkafka"]
+	path = plugins/kafka/librdkafka
+	url = https://github.com/edenhill/librdkafka.git
```

Comment:
We should fork this repo into ours (I'll do that) and then reference ours in the git submodule - that way we are not relying on the foreign repo staying around...